prusti-interface/src/environment/mod.rs

```diff
@@ -493,14 +493,7 @@ impl<'tcx> Environment<'tcx> {
     }
 
     pub fn type_is_allowed_in_pure_functions(&self, ty: ty::Binder<'tcx, ty::Ty<'tcx>>, param_env: ty::ParamEnv<'tcx>) -> bool {
```

Comment:
Actually let's just use `type_is_copy` directly then :)

Reply:
I kind of liked the additional method, but I'm ok with removing it.